ui/src/shared/components/Dygraph.js

```diff
@@ -45,13 +45,18 @@ class Dygraph extends Component {
       isGraphFilled: fillGraph,
       isBarGraph,
       options,
+      labels,
     } = this.props
 
     const timeSeries = this.timeSeries
     const graphRef = this.graphRef
 
     let defaultOptions = {
+      ...options,
+      labels,
```

Comment:
does this help fix the bug? or what is this being added for? looking at the dygraphs docs, it looks like this is about labels for the time series themselves, and `ylabel` below is the option that seems pertinent to this bug.

Reply:
project board

Reply:
👍